tests/cases/standalone/common/types/json/json.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE jsons (j JSON, t timestamp time index);
+
+--Insert valid json strings--
+INSERT INTO jsons VALUES('{"a":1, "b":2, "c":3}', 1), ('{"a":1, "b":2, "c":3, "d":4}', 2), ('{"a":1, "b":2, "c":3, "d":4, "e":5}', 3);
+
+SELECT * FROM jsons;
+
+--Insert invalid hex strings--
+DELETE FROM jsons;
+
+INSERT INTO jsons VALUES('\xaa\xff\xaa'::BYTEA, 1), ('\xaa\xff\xaa\xaa\xff\xaa'::BYTEA, 2), ('\xaa\xff\xaa\xaa\xff\xaa\xaa\xff\xaa'::BYTEA, 3);
+
+SELECT * FROM blobs;
```

Comment:
is this `blobs` supposed to be `jsons`?

Reply:
Yes, I misspelled here qaq. Changed already (actually removed this test owo)